configs/mainnet.yaml

```diff
@@ -144,14 +144,26 @@ ATTESTATION_SUBNET_PREFIX_BITS: 6
 # Deneb
 # `2**7` (=128)
 MAX_REQUEST_BLOCKS_DENEB: 128
-# MAX_REQUEST_BLOCKS_DENEB * MAX_BLOBS_PER_BLOCK
-MAX_REQUEST_BLOB_SIDECARS: 768
 # `2**12` (= 4096 epochs, ~18 days)
 MIN_EPOCHS_FOR_BLOB_SIDECARS_REQUESTS: 4096
 # `6`
 BLOB_SIDECAR_SUBNET_COUNT: 6
-## `uint64(6)`
+# `uint64(6)`
 MAX_BLOBS_PER_BLOCK: 6
+# MAX_REQUEST_BLOCKS_DENEB * MAX_BLOBS_PER_BLOCK
+MAX_REQUEST_BLOB_SIDECARS: 768
+
+# Electra
+# 2**7 * 10**9 (= 128,000,000,000)
+MIN_PER_EPOCH_CHURN_LIMIT_ELECTRA: 128000000000
+# 2**8 * 10**9 (= 256,000,000,000)
+MAX_PER_EPOCH_ACTIVATION_EXIT_CHURN_LIMIT: 256000000000
+# `9`
+BLOB_SIDECAR_SUBNET_COUNT_ELECTRA: 9
```

Comment:
Could we add the variable `TARGET_BLOBS_PER_BLOCK_ELECTRA: 6` here?

Reply:
I'd prefer to have `TARGET_BLOBS_PER_BLOCK: 3` and `TARGET_BLOBS_PER_BLOCK_ELECTRA: 6`

Reply:
`TARGET_BLOBS_PER_BLOCK` CL param is introduced in Electra, so it can be named `TARGET_BLOBS_PER_BLOCK_ELECTRA`, but `TARGET_BLOBS_PER_BLOCK: 3` definition on CL side is a bit odd

Reply:
Historically, when introducing a new config variable it doesn't include the fork name. So just `TARGET_BLOBS_PER_BLOCK` would be the most appropriate, but yeah it feels weird.

Reply:
Added a new `TARGET_BLOBS_PER_BLOCK_ELECTRA` config variable.